components/webgl/webgl_thread.rs

```diff
@@ -368,8 +368,8 @@ impl WebGLThread {
                     }))
                     .unwrap();
             },
-            WebGLMsg::ResizeContext(ctx_id, size, sender, epoch) => {
-                let _ = sender.send(self.resize_webgl_context(ctx_id, size, epoch));
+            WebGLMsg::ResizeContext(ctx_id, size, sender) => {
+                let _ = sender.send(self.resize_webgl_context(ctx_id, size));
```

Comment:
Hm, this can be a problematic, as we will not await canvas clearing in compositor (if there is no rAF that changes it's content afterwards). We could migrate this by handling clearing fully in script (returning None instead of image key).